src/App.scss

```diff
@@ -1 +1,93 @@
-// not empty
+@import './styles/mixins';
```

Comment:
Please, add all files (not only src folder) to git (including package.json):

![image](https://github.com/mate-academy/react_phone-catalog/assets/86923089/1504e9cf-3d16-4d06-a8a1-d025ecd0f023)